telegram/_bot.py

```diff
@@ -5554,12 +5632,21 @@ async def create_new_sticker_set(
         user_id: Union[str, int],
         name: str,
         title: str,
-        emojis: str,
+        # Deprecated params since bot api 6.6
+        # ----
```

Comment:
```suggestion
        # <----
```

similar elsewhere. Makes it more obvious that this marks the start/end of something